tutorials/neurips_challenge_quickstart.md

```diff
@@ -85,9 +98,9 @@ pip install -r requirements.txt tokenizers sentencepiece
 
 ## Downloading Model Checkpoints
 
-This section explains how to download the StableLM 3B Base model, the smallest model supported in Lit-GPT. (The downloaded and converted checkpoints will occupy approximately 28 Gb of disk space.)
+This section explains how to download the StableLM 3B Base model, one of the smallest models supported in Lit-GPT (except for Pythia). The downloaded and converted checkpoints will occupy approximately 28 Gb of disk space.
```

Comment:
My English is pretty bad, so maybe that why I don't understand it, but for me it sounds like Pythia is not supported by Lit-GPT. 
I tried couple of translaters and they are also confused 😄.

Reply:
ok ok, fixed it :)